### ISSUE TITLE
Unable to get full queries of objects through the API without paging

### ISSUE THREAD

**Issue by shots47s** [aces/cbrain#789]
Currently, if you query returns more than the allowed number of objects (which appears to be set at 50), there is no way to continue to get the rest of the query.  We should implement some way to either request further pages or a continuation token so that the client can make subsequent calls to get all of the useful data.

**Comment by prioux** [comment]
The good thing is, it's already fully supported in the controllers with the params `page` and `per_page`; we simply need to add them to the YAML and JSON description of the API.

I suggest in-query params, e.g.

```yaml
        - name: page
          in: query
          type: integer
          format: int64
          description: Page number
          required: false
```

**Comment by MontrealSergiy** [comment]
@prioux suggestion seems trivial , should I? In ideal world api should be able to either return number of items (perhaps in the same or another endpoint).

**Comment by prioux** [comment]
We don't need a new plugin, like I explained, it's already implemented in the controllers.

**Comment by prioux** [comment]
Fixed by #811
